control-spec.txt

```diff
@@ -2098,6 +2107,19 @@
    actual reason given by the remote OR for closing the circuit. Clients MUST
    accept reasons not listed above.  Reasons are as listed in tor-spec.txt.
 
+   The "UNUSABLE_FOR_NEW_CONNS" field is provided only in versions 0.4.X and
+   later. UNUSABLE_FOR_NEW_CONNS is a boolean digit set to 1 if no new
+   connection can be opened on the circuit or 0 if further connections can be
+   opened.
+
+   The "IDLE_TIMEOUT" field is provided only in versions 0.4.X and later.
+   IDLE_TIMEOUT describes the number of seconds before the circuit will be
+   closed if it remains idle, once built.
+
+   The "REMAINING_RELAY_EARLY_CELLS" field is provided only in versions 0.4.X
+   and later. REMAINING_RELAY_EARLY_CELLS is a value from 0 to 8 and represents
```

Comment:
We shouldn't say that this is a value from 0 to 8: we should say it is an integer.  Future versions of Tor might pick a starting value other than 8.

Also, we should be clear that this is the number of RELAY_EARLY commands that we can still _send_, so that it doesn't get confused with a related counter in OR circuits that counts how many RELAY_EARLY commands we can still _receive_.

Reply:
I updated the documentation to `REMAINING_RELAY_EARLY_CELLS is an integer value representing the number of RELAY_EARLY commands still available to send on this circuit`.

Does that seem precise enough?